src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -219,6 +219,23 @@ private static TestInputConfiguration parse(String inputFilePath,
         return testInputConfigBuilder.build();
     }
 
+    public static TestInputConfiguration parseGoogleConfig(String inputFilePath)
+            throws Exception {
+        final TestInputConfiguration.Builder testInputConfigBuilder =
+                new TestInputConfiguration.Builder();
+        final Path filePath = Paths.get(inputFilePath);
+        final List<String> lines = readFile(filePath);
+
+        try {
+            setViolationsForGoogleConfig(testInputConfigBuilder, lines);
+        }
+        catch (CheckstyleException ex) {
+            throw new CheckstyleException(ex.getMessage() + " in " + inputFilePath, ex);
+        }
+
+        return testInputConfigBuilder.build();
```

Comment:
there is no config in ITs, you should return only violations, this is exactly what you need, and method should do only this.
fact that you reuse TestInputConfiguration.Builder is internal details that shuold not be leaked outside of this class.

Reply:
done

https://github.com/checkstyle/checkstyle/blob/a0f82691156c959fbb7dccd991c3e5c4f83022e1/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L235-L238